frameworks/cassandra/docs/api-reference.md

```diff
@@ -4,12 +4,7 @@ navigationTitle:
 excerpt:
 title: API Reference
 menuWeight: 70
-
-packageName: beta-cassandra
-serviceName: cassandra
 ---
+{% assign data = site.data.services.cassandra %}
```

Comment:
It isn't possible to centralize this assignment is it?

Reply:
Not AFAICT, the scopes appear to just be:
- site-wide
- the current page
No separate 'section' scope.